src/ASM/ASMbase.h

```diff
@@ -576,6 +577,11 @@ class ASMbase
   //! \brief Returns the number of projection nodes for this patch.
   virtual size_t getNoProjectionNodes() const { return this->getNoNodes(1); }
 
+  //! \brief Returns a field using the projection basis.
+  //! \param[in] coefs The coefficients for the field
+  virtual Fields* getProjectedFields(const Vector& coefs, size_t nf) const
```

Comment:
Make arguments anonymous on this interface and remove the coefs doxy also.

Reply:
done.